src/ImageSharp/Processing/Processors/CloningImageProcessor{TPixel}.cs

```diff
@@ -45,16 +47,12 @@ protected CloningImageProcessor(Image<TPixel> source, Rectangle sourceRectangle)
         protected Configuration Configuration { get; }
 
         /// <inheritdoc/>
-        public Image<TPixel> CloneAndApply()
+        public Image<TPixel> CloneAndExecute()
```

Comment:
Suspicious: two verbs in the method name.

Reply:
Naming is hard. I am up for suggestions.

Reply:
It's not the naming actually. Might be a sign, that we can improve the design here.